daemon/main.c

```diff
@@ -1339,6 +1339,27 @@ int main(int argc, char **argv) {
 
     send_statistics("START", "-",  "-");
 
+    // ------------------------------------------------------------------------
+    // Report ACLK build failure
+#ifndef ENABLE_ACLK
+    error("This agent doesn't have ACLK.");
+    char filename[FILENAME_MAX + 1];
+    snprintfz(filename, FILENAME_MAX, "%s/.aclk_report_sent", netdata_configured_varlib_dir);
```

Comment:
I think we need to remove the file and send everytime (just as we do with START/ EXIT)

The file is never removed (as far as I can see) even if there is a successful build.

If we have a successful build followed by a failed one (say a month later) we want to know that.

Reply:
Hmmm, well in such case we might want to get product involved as the task seems to be different.

Reply:
No this does not seem to be correct. We want to know where installation failed because the dependency was unavailable. The case where it worked, but the user is recompiling it in the same source tree and it then fails does not seem to be relevant to the ask. What they care about is that if somebody tries to install it and the ACLK can't be enabled then we find out about it.

Reply:
I understand the ask as being per-installation rather than per-compile.

Reply:
then we have to delete it from installer

Reply:
I don't think that it needs to be deleted at all - if the failure has been reported then there is no need to report it again from the same machine.